http/check_service.go

```diff
@@ -679,3 +683,209 @@ func (h *CheckHandler) handleDeleteCheck(w http.ResponseWriter, r *http.Request)
 
 	w.WriteHeader(http.StatusNoContent)
 }
+
+func checkIDPath(id influxdb.ID) string {
+	return path.Join(prefixChecks, id.String())
+}
+
+// CheckService is a client to the handlers in this package.
+// It does not implement influxdb.CheckService because it returns a concrete representation of the API response
+// and influxdb.Check as returned by that interface is not appropriate for this use case.
+type CheckService struct {
+	Client *httpc.Client
+}
+
+func (s *CheckService) FindCheckByID(ctx context.Context, id influxdb.ID) (*Check, error) {
+	span, _ := tracing.StartSpanFromContext(ctx)
+	defer span.Finish()
+
+	var cr Check
+	err := s.Client.
+		Get(checkIDPath(id)).
+		DecodeJSON(&cr).
+		Do(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	return &cr, nil
+}
+
+func (s *CheckService) FindCheck(ctx context.Context, filter influxdb.CheckFilter) (*Check, error) {
```

Comment:
comment for what this does? just returns the first Check found according to a filter?